### PR TITLE
DOC: ophyd device component helper

### PR DESCRIPTION
**Pull request by klauer**
(Current master for comparison: https://pcdshub.github.io/pcdsdevices/generated/generated/pcdsdevices.attenuator.AttenuatorCalculator_AT2L0.html)

Attempt 2; I think this looks much better:

image
image

Retains the previous method-documenting functionality, but pares it down significantly:

image

### CODE REVIEW

**Comment by klauer** [comment]
Only issue so far is the size of the table and the need to scroll right to see it all. Modifying the CSS you can get the comprehensive but ugly:

image

**Comment by ZLLentz** [comment]
I think this looks very good

File: docs/source/conf.py [hunk 1]
```diff
@@ -176,7 +183,7 @@
 #  dir menu entry, description, category)
 texinfo_documents = [
     (master_doc, 'PCDSDevices', 'PCDS Devices Documentation',
-     author, 'PCDSDevices', 'One line description of project.',
```
Comment (ZLLentz) [inline docs/source/conf.py:179]: Oops

**Review by ZLLentz — APPROVED** [review]

**Comment by ZLLentz** [comment]
Ready to merge?

**Comment by klauer** [comment]
@ZLLentz if you're OK with the blacklist of methods/attributes as-is, I think this is good to go.

**Comment by ZLLentz** [comment]
Yes, I think the blacklists are good

**Comment by klauer** [comment]
Doc building failed on travis due to a frustrating error: 

https://travis-ci.com/github/pcdshub/pcdsdevices/jobs/386412558

**Comment by ZLLentz** [comment]
That's super annoying...

**Pull request merged**